polls/impl/src/java/org/sakaiproject/poll/service/impl/PollListManagerImpl.java

```diff
@@ -546,7 +546,18 @@ public String[] myToolIds()
 		}
 
 
-		public Map<String, String> transferCopyEntities(String fromContext, String toContext, List<String> resourceIds, List<String> transferOptions, boolean condition) {
+		public Map<String, String> transferCopyEntities(String fromContext, String toContext, List<String> resourceIds, List<String> transferOptions, boolean cleanup) {
+			if (cleanup) {
+                try {
+                    // Find and delete any polls from the destination site
+                    List<Poll> toPolls = findAllPolls(toContext);
+                    for (Poll poll : toPolls) {
+                        deletePoll(poll);
+                    }
+                } catch(Exception e) {
+                    log.error(e.getMessage(), e);
+                }
```

Comment:
Please fix spacing issues, thanks!

Reply:
what's the spacing issue? tabs?

Reply:
It's just polls, man, just polls ... spacing all over the shop

Reply:
i see spaces vs tabs. i went in and edited

Reply:
Thanks, it looks fine now!